src/index.ts

```diff
@@ -178,28 +171,23 @@ function successLoader(
  */
 function reloadRootFileNamesFromConfig(
   loaderContext: webpack.loader.LoaderContext,
-  instance: TSInstance
+  instance: TSInstance,
+  options: LoaderOptions
```

Comment:
`instance.loaderOptions !== options`, ouch 😖 Pretty sure we do this wrong in lots more places, but this one was needed to get tests passing